### PR TITLE
fix(assert): fix asserts css to elements visible and invisible

### PR DESCRIPTION
**Pull request by xXNoceboXx**

### CODE REVIEW

File: src/ExtraSessionTrait.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ public function waitForSeconds($seconds)
      */
     public function iWaitForCssElementBeingVisible($element, $seconds)
     {
-        $result = $this->getSession()->wait($seconds * 1000, sprintf("$('%s').length >= 1", $element));
+        $result = $this->getSession()->wait($seconds * 1000, sprintf("$('%s').length >= 1 && $('%s').css('display') != 'none'", $element, $element));
```
Comment (mremi) [inline src/ExtraSessionTrait.php]: Please use `%1$s` to avoid repeating `$element`

**Review by mremi — COMMENTED** [review]
Please update CHANGELOG as well

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,3 +6,4 @@ master
 
 * Allow symfony/stopwatch ^2.8
 * Fixed iWaitForCssElementBeingVisible / iWaitForCssElementBeingInvisible
+* Fixed iWaitForCssElementBeingVisible / iWaitForCssElementBeingInvisible to check css attribute display
```
Comment (mremi) [inline CHANGELOG.md]:
```markdown
Fixed iWaitForCssElementBeingVisible / iWaitForCssElementBeingInvisible:
* throw \RuntimeException to make test fail
* make sure element is visible or not
```

**Comment by mremi** [comment]
LGTM @BitScout @bmaziere ok for you?

**Review by mazsudo — APPROVED** [review]

**Comment by BitScout** [comment]
Do you have experience with checking for size to find out the actual visibility? The way it's done now, the parent might be hidden.

`var isVisible = element.offsetWidth > 0 || element.offsetHeight > 0;`

https://stackoverflow.com/questions/1343237/how-to-check-elements-visibility-via-javascript

**Comment by xXNoceboXx** [comment]
@BitScout I think just check display it's correct because to use selector css and you can select parent :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,4 +5,6 @@ master
 ------
 
 * Allow symfony/stopwatch ^2.8
-* Fixed iWaitForCssElementBeingVisible / iWaitForCssElementBeingInvisible
+Fixed iWaitForCssElementBeingVisible / iWaitForCssElementBeingInvisible:
+* throw \RuntimeException to make test fail
```
Comment (mremi) [inline CHANGELOG.md]: please add 2 whitespaces before `*` to fix indentation

**Comment by BitScout** [comment]
Okay for me.

**Pull request merged**